recipes-extended/tmux/files/tmux.conf

```diff
@@ -0,0 +1,2 @@
+set -g lock-command vlock
+bind X lock-session
```

Comment:
I had a thought while we were talking about sourcing conf files on Teams. Does the tmux `source` directive work with wildcards? If it does, maybe we can make this even better by having the tmux package install an empty `/usr/share/tmux/conf.d` directory. Then this OE conf file can say `source /usr/share/tmux/conf.d/*`, and the `nilrt-snac.ipk` can just install the remaining timeout snippet into a file it owns there without having to write anything into the `/etc/tmux.conf`. What do you think?

Reply:
Updated to create and source conf files from /usr/share/tmux/conf.d/